data-science-onramp/feature-engineering/feature_engineering.ipynb

```diff
@@ -1,4795 +1,1019 @@
 {
```

Comment:
<p>Second sentence:</p><p> </p><p>Original:</p><p>Similar to loading the Gas Prices dataset, this query selects the columns needed ...</p><p> </p><p>Proposed:</p><p><strong>Similarly</strong> to loading the Gas Prices dataset, this query selects the columns needed</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/leahecole/python-docs-samples/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Reply:
<p>Done, but omg it sounds so awkward English is terrible.</p>